config/config.yaml

```diff
@@ -31,3 +31,5 @@ OperatorPrivateKey:
 bootnode:
   ExternalIP:
   PrivateKey:
+
```

Comment:
@nivBlox doesn't seem to be relevant to this PR

Reply:
its pretty code